### ISSUE TITLE
No margin on top since 0.117

### ISSUE THREAD

**Issue by m4dm4rtig4n** [thomasloven/lovelace-layout-card#98]
Hi,

Since I updated to 0.117, I no longer have space between the header and my cards when first card is a layout-card :/

Screen: 
![image](https://user-images.githubusercontent.com/7872518/97632102-531c9080-1a32-11eb-84e0-4ea3c35220ee.png)

To reproduce : 
```
path: default_view
panel: true
cards:

  - type: custom:layout-card
    layout: vertical
    max_columns: 3
    cards:
      - type: horizontal-stack
        cards:
          - type: picture-entity
            style: |
              ha-card {
              {% if states(config.entity) == "home" %}
                filter: brightness(100%);
              {% else %}
                filter: brightness(30%);
              {% endif %}
              }
            show_name: false
            entity: person.clement
            image: /local/clement.png
            show_state: false
          - type: picture-entity
            style: |
              ha-card {
              {% if states(config.entity) == "home" %}
                filter: brightness(100%);
              {% else %}
                filter: brightness(30%);
              {% endif %}
              }      
```

**Comment by Jeoffreybauvin** [comment]
Same issue here.

![Screenshot_1](https://user-images.githubusercontent.com/701928/97724463-3c774780-1acd-11eb-846a-e90d16af78ae.png)

**Comment by thomasloven** [comment]
This should be fixed with 2.0.0.
Please confirm.
